src/views/steps/subviews/service.coffee

```diff
@@ -0,0 +1,83 @@
+{ItemView} = require 'backbone.marionette'
+_ = require 'underscore'
+
+ModalView = require '../../modal'
+
+module.exports = class ServiceView extends ItemView
+    template: require '../../templates/service'
+
+    className: 'service'
+
+    ui:
+        connect: '[role=connect]'
```

Comment:
`role=connect` is a valid aria role ? I can’t find it in https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques. Maybe I misinterpret this form.